tests/msg_avail/main.c

```diff
@@ -34,28 +36,27 @@ int main(void)
 
     msg_init_queue(msg_queue, MSG_QUEUE_LENGTH);
 
+    puts("[START]");
```

Comment:
mhm, yeah timing test might be a special thing to consider.  Still, currently many tests make excessive use of `puts` or `printf` which makes it impossible to fit those on constraint boards, i.e., like arduino uno or duemilanove with 2K RAM. And in most cases these `puts` could be replaced by a proper readme, or make output optional by using `LOG_<something>` instead.